index.js

```diff
@@ -1,12 +1,15 @@
-var exporter = {}
+var exporter = {};
 
-var chooseRandomNumer = function(startindex, endIndex){
+var chooseRandomNumer = function (startindex, endIndex) {
   return Math.floor(Math.random() * (startindex - endIndex)) + endIndex;
 }
 
-exporter.getRandomNumberInRangeSync = function(array, count, startIndex, endIndex) {
-  if(startIndex < 0 || startIndex > array.length - 1) {
-    throw new Error('Invalid startIndex value'); 
+exporter.getRandomNumberInRangeSync = function (array, count, startIndex, endIndex) {
+  if (Array.isArray(array) == false) {
+    throw new Error('Invalid array object');
+  }
+  if (startIndex < 0 || startIndex > array.length - 1) {
+    throw new Error('Invalid startIndex value');
```

Comment:
Do you find the need to print startIndex, endIndex, count values for debugging purpose??